src/main/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActor.scala

```diff
@@ -197,6 +199,7 @@ private class TaskLauncherActor(
               await(instanceTracker.forceExpunge(instance.instanceId)): @silent
               await(instanceTracker.schedule(Instance.scheduled(instance.runSpec)))
             }
+            Status.Success(s"Rescheduled ${instance.instanceId} due to provision timeout.")
           } pipeTo self
```

Comment:
why do we even pipeTo self the success? does it make any sense? I would just pipe to self errors if anything...

Reply:
Fair point. We could use `onComplete` to send the error or just log it and send nothing.

Reply:
Yeah, that's a fair point :)